django/contrib/admin/widgets.py

```diff
@@ -6,12 +6,12 @@
 import copy
 
 from django import forms
-from django.contrib.admin.templatetags.admin_static import static
 from django.core.urlresolvers import reverse
 from django.db.models.deletion import CASCADE
 from django.forms.utils import flatatt
 from django.forms.widgets import RadioFieldRenderer
 from django.template.loader import render_to_string
+from django.templatetags.static import static
```

Comment:
Shouldn't we be able to remove usage of `static()` in this file now that `absolute_path()` adds it?